src/main/io/osd.c

```diff
@@ -3915,231 +3944,249 @@ static void osdUpdateStats(void)
     stats.max_altitude = MAX(stats.max_altitude, osdGetAltitude());
 }
 
-static void osdShowStatsPage1(void)
+static void osdShowStats(bool isSinglePageStatsCompatible, uint8_t page)
 {
     const char * disarmReasonStr[DISARM_REASON_COUNT] = { "UNKNOWN", "TIMEOUT", "STICKS", "SWITCH", "SWITCH", "KILLSW", "FAILSAFE", "NAV SYS", "LANDING"};
-    uint8_t top = 1;    /* first fully visible line */
+    uint8_t top = 1;  // Start one line down leaving space at the top of the screen. 
+    size_t multiValueLengthOffset = 0;
+
     const uint8_t statNameX = osdDisplayIsHD() ? 11 : 1;
     const uint8_t statValuesX = osdDisplayIsHD() ? 30 : 20;
     char buff[10];
-    statsPagesCheck = 1;
+
+    if (page > 1)
+    page = 0;
```

Comment:
Missing the indentation here.

Reply:
I corrected this indentation. As RC2 was released today, I'm getting the impression that having this be part of 6.0 isn't going to happen at this point anyways.  I may close this PR and revisit it at a later date if someone else doesn't take up the reigns in a later version. Keeping up with this has been a bit more involved than I bargained for.

Reply:
Keep the PR open. It's an update, rather than a new feature. So it still could make it in to 6.0. If not, it will be merged in to 6.1.